pkg/acsengine/testdata/windows/kubernetes_expected_params.json

```diff
@@ -32,11 +32,38 @@
   "kubeConfigPrivateKey": {
     "value": "a3ViZUNvbmZpZ1ByaXZhdGVLZXk="
   },
+  "kubectlDownloadURL": {
+    "value": "https://storage.googleapis.com/kubernetes-release/release/v1.5.3/bin/linux/amd64/kubectl"
+  },
   "kubectlVersion": {
     "value": "v1.5.3"
   },
+  "kubernetesAddonManagerSpec": {
+    "value": "gcr.io/google_containers/kube-addon-manager-amd64:v5.1"
+  },
+  "kubernetesAddonResizerSpec": {
+    "value": "gcr.io/google_containers/addon-resizer:1.6"
+  },
+  "kubernetesDNSMasqSpec": {
+    "value": "gcr.io/google_containers/kube-dnsmasq-amd64:1.3"
+  },
+  "kubernetesDashboardSpec": {
+    "value": "gcr.io/google_containers/kubernetes-dashboard-amd64:v1.5.1"
+  },
+  "kubernetesExecHealthzSpec": {
+    "value": "gcr.io/google_containers/exechealthz-amd64:1.2"
+  },
+  "kubernetesHeapsterSpec": {
+    "value": "gcr.io/google_containers/heapster:v1.2.0"
+  },
   "kubernetesHyperkubeSpec": {
-    "value": "gcr.io/google_containers/hyperkube-amd64:v1.5.3"
```

Comment:
Why is this change here? Is this rebased? Or was there a rebase problem that reverted from 1.5.3 to 1.5.1?